content/articles/2018-11-tidymodels-updates.Rmarkdown

```diff
@@ -0,0 +1,163 @@
+
+There are also three new vignettes. One has already been mentioned that describes [multiclass averaging](https://tidymodels.github.io/yardstick/articles/multiclass.html). The other two focus on the three main [metric types](https://tidymodels.github.io/yardstick/articles/metric-types.html) in yardstick, and on implementing [custom metrics](https://tidymodels.github.io/yardstick/articles/custom-metrics.html) for personal use.
+
+## **recipes**
+
+### Breaking changes
+
+One big change was to make the argument names more consistent with the tidyverse standards and to also make them consistent with `dials` and other packages. For example, [`step_pca()`](https://tidymodels.github.io/recipes/reference/step_pca.html) now has an argument `num_comp` that replaces the previous `num` argument. This will pay off later when we enable the detection of tuning parameters and the automatic determination of grid values or parameter ranges. The biggest name change is in [`bake()`](https://tidymodels.github.io/recipes/reference/bake.html); `newdata` is now `new_data`. For the time being, a warning will be issued when `newdata` is used but that won't last past the next version. The list of name changes are detailed [here](https://tidymodels.github.io/recipes/news/index.html). 
+
+In recipes, variables can have different roles (e.g. "predictor" or "outcome"). Beyond those set by the package, roles are largely user specified and can be pretty much anything. Previously, only a single role was allowed. The new version of recipes expands the number of roles per column. This now means that [`add_role()`](https://tidymodels.github.io/recipes/reference/roles.html) will _append_ roles, and the new function [`update_role()`](https://tidymodels.github.io/recipes/reference/roles.html) will reset them. It also changes how the `summary()` results for a recipe are returned since there can now be multiple rows per column variable. 
+
+A feature that we will be working on in the _next_ version is to be able to reference (and use) previous steps. For example, if you center some variables, you might want to uncenter them at a later step. For this future feature, this version of `recipes` mandates an ID field for each step. The ID can be anything, but the current convention is to use the step name followed by random digits (e.g. `"center_irqtH"`). 
```

Comment:
is to be able to ⇨ "the ability to reference…" 
(matching noun for "A feature")